sitl_setup/run.sh

```diff
@@ -1,9 +1,27 @@
 #!/bin/bash
+# Make sure line endings are LF for this file!!!
```

Comment:
Is this worth highlighting in readme?

Reply:
Don't think so tbh because `run.sh` should hardly ever need to be updated

Reply:
Aight sounds good